### DISCUSSION TITLE
Http admin question.

### DISCUSSION THREAD

**Discussion by kisisten** [rabbitmq/rabbitmq-server#4244] [Other] [answered]
Hi guys, we have a single node rabbitmq running version 3.7.11 in out-of-the box mode, without environment variables nor configurations files, configured only with rabbitmqctl.
It has the web UI management module enabled which is running on default port {http,15672,"::"}

Can I use rabbitmqctl to disable the default port 15672 and enable ssl on 15671 plus all the certificates that are required, or I would need to use a config file for this purpose?
Thanks.

**Comment by lukebakken** [comment, marked as answer]
You must use configuration to enable TLS:

https://www.rabbitmq.com/management.html#single-listener-https

**Reply by kisisten** [reply]
The one thing I can figure out is, if I already have listeners/queues/topics configured via rabbitmqctl, how do I migrate this environment to use config files on 3.7.11?
Thanks.

**Reply by lukebakken** [reply]
If your queues are durable, they will persist after restarting RabbitMQ. Stop RabbitMQ, add your configuration, and restart.

Your applications should reconnect and redeclare their topology if they have been written to do so.

**Reply by lukebakken** [reply]
Ensure your messages have been published with the `persistent` flag set, of course.

**Reply by kisisten** [reply]
Thank you for you help. I was able to figure it out.
